docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.9'
 
 # volumes are not in common, because phpstorm doesn't understand it
 x-common: &common
-  image: ghcr.io/myparcelnl/php-xd:7.2
+  image: ghcr.io/myparcelnl/php-xd:8.2
```

Comment:
Ditto als mijn opmerking in de test workflow, alleen dan toepasbaar op lokale ontwikkeling impact.

Reply:
zelfde antwoord